### PR TITLE
pscanrules: Update reference links (http -> https) Alerts: 90011 and 10035

### PR DESCRIPTION
**Pull request by 8linkz**
## Overview
I have only replaced http with https, the actual link has remained the same.
https://www.zaproxy.org/docs/alerts/90011/ and https://www.zaproxy.org/docs/alerts/10035/

## Related Issues
https://github.com/zaproxy/zaproxy/issues/8262

## Checklist
- [ ] Update help
- [x] Update changelog
- [ ] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by thc202** [comment]
The rule 10035 should be updated in this PR too.

**Comment by 8linkz** [comment]
> The rule 10035 should be updated in this PR too.

done please check

**Review by kingthorin — APPROVED** [review]

File: addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties [hunk 1]
```diff
@@ -282,35 +282,35 @@ pscanrules.serverheaderversioninfoleak.name = Server Leaks Version Information v
 
 pscanrules.stricttransportsecurity.compliance.malformed.content.desc = A HTTP Strict Transport Security (HSTS) header was found, but it contains some content that was not expected (perhaps curly quotes), the expectation is that the content be printable ASCII characters.
 pscanrules.stricttransportsecurity.compliance.malformed.content.name = Strict-Transport-Security Malformed Content (Non-compliant with Spec)
-pscanrules.stricttransportsecurity.compliance.malformed.content.refs = http://tools.ietf.org/html/rfc6797
+pscanrules.stricttransportsecurity.compliance.malformed.content.refs = https://tools.ietf.org/html/rfc6797
```
Comment (thc202) [inline addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties:285]: This is a 301, better update to the final URL.

Comment (8linkz) [reply addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties]: > This is a 301, better update to the final URL.

done for all affected links

Comment (thc202) [reply addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties]: This one was not changed yet https://github.com/zaproxy/zap-extensions/pull/5201/files#diff-5f76e38a9cd7b2a22adb1ee21b0d3c550e7251609be7879e71d209cebecaed86R313

File: addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties [hunk 2]
```diff
@@ -282,35 +282,35 @@ pscanrules.serverheaderversioninfoleak.name = Server Leaks Version Information v
...
-pscanrules.stricttransportsecurity.plain.resp.refs = http://tools.ietf.org/html/rfc6797#section-8.1
+pscanrules.stricttransportsecurity.plain.resp.refs = https://datatracker.ietf.org/doc/html/rfc6797#section-8.1
 pscanrules.stricttransportsecurity.plain.resp.soln = Review the configuration of this control. Ensure that your web server, application server, load balancer, etc. is configured to set Strict-Transport-Security for HTTPS responses.
-pscanrules.stricttransportsecurity.refs = https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Strict_Transport_Security_Cheat_Sheet.html\nhttps://owasp.org/www-community/Security_Headers\nhttp://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security\nhttp://caniuse.com/stricttransportsecurity\nhttp://tools.ietf.org/html/rfc6797
+pscanrules.stricttransportsecurity.refs = https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Strict_Transport_Security_Cheat_Sheet.html\nhttps://owasp.org/www-community/Security_Headers\nhttps://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security\nhttps://caniuse.com/stricttransportsecurity\nhttps://tools.ietf.org/html/rfc6797
```
Comment (thc202) [inline addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties:313]:
```suggestion
pscanrules.stricttransportsecurity.refs = https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Strict_Transport_Security_Cheat_Sheet.html\nhttps://owasp.org/www-community/Security_Headers\nhttps://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security\nhttps://caniuse.com/stricttransportsecurity\nhttps://datatracker.ietf.org/doc/html/rfc6797
```

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**